src/core/raster/qgshillshaderenderer.h

```diff
@@ -102,11 +108,18 @@ class CORE_EXPORT QgsHillshadeRenderer : public QgsRasterRenderer
      */
     void setZFactor( double zfactor ) { mZFactor = zfactor; }
 
+    /**
+     * @brief Set IsMultiDirection
+     * @param isMultiDirection Is multi direction
+     */
+    void SetMultiDirection( bool isMultiDirection ){ mMultiDirection = isMultiDirection; }
```

Comment:
Should be `setMultiDirection`. But perhaps `setMultiDirectional` is a better name?